src/components/Modal/tests/Modal.test.tsx

```diff
@@ -1,244 +0,0 @@
-import * as React from 'react';
```

Comment:
@dheeraja00 You just deleted all the unit tests.  I hope you're planning to rewrite them.